README.md

```diff
@@ -0,0 +1,106 @@
+
+ # Mopro React Native Package
+ 
+ This library exposes the native Rust bindings generated by [Mopro](https://github.com/zkmopro/mopro). You get cross-platform mobile proving and verification with a slim, type‑safe JS/TS API.
+ 
+ > Mopro handles circuit compilation, Rust ↔︎ mobile bindings, and multi‑proof back‑ends so you can focus on building privacy‑preserving features.
+
+ 
+ ## How to Import the Package
+ Use a Node.js package manager in your React Native app to install dependencies. For example:
+ ```sh
+ # npm
+ npm install https://github.com/zkmopro/mopro-react-native-package
+ # yarn / pnpm
+ yarn add https://github.com/zkmopro/mopro-react-native-package
+ ```
+
+ Alternatively, you can manually add it to your package.json:
+ ```json
+ "dependencies": {
+  "mopro-react-native-package": "github:zkmopro/mopro-react-native-package",
+}
+```
+
+## How to Use the Package
+Here is an example of how to integrate and use this package
+
+```ts
+ import {
+   generateProof,
+   verifyProof,
+   ProofLib,          // enum: 'arkworks' | 'rapidsnark' | ...
+ } from 'mopro-react-native-package';
+ 
+ async function main() {
+   const zkey = `${RNFS.DocumentDirectoryPath}/multiplier2_final.zkey`;
+   const inputs = { a: ['3'], b: ['5'] }; // 3 × 5 = 15
+ 
+   const { proof, publicInputs } = await generateProof({
+     zkeyPath: zkey,
+     inputs: JSON.stringify(inputs),
+     proofLib: ProofLib.arkworks,
+   });
```

Comment:
I think the API is wrong
https://github.com/zkmopro/mopro-react-native-package/blob/388a42e790500e510d8ccb201af38a89e00590d2/example/App.tsx#L36
or we need to update the package